src/sentry/migrations/0130_dashboard_delete_object_status.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-11-19 22:26
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    atomic = True
+
+
+    dependencies = [
+        ('sentry', '0129_remove_dashboard_keys'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Dropping fields like this could cause an outage if the model gets any read traffic. While it is not likely with this model, you should keep this in mind for other schema changes.

Typically what we do is use two migrations:

1. The first migration uses `SeparateDatabaseAndState` migrations to remove the field from Django's metadata. You would also remove the field from the model at this point.
2. Deploy the change from 1.
3. Deploy another migration that uses RunSQL to drop the column. Because django no longer knows about this field it is safe to drop as it won't affect read queries.

Reply:
For step 3, I created this PR https://github.com/getsentry/sentry/pull/22172